src/schema.ts

```diff
@@ -1,7 +1,21 @@
 import { z } from 'zod';
 
-export const oktaAPIError = z.object({
+export const oktaAPIErrorSchema = z.object({
   status: z.number(),
 });
-
-export type OktaAPIError = z.infer<typeof oktaAPIError>;
+const urlStart = 'https://';
+const urlEnd = '.okta.com';
+export const urlSchema = z
+  .string()
+  .url()
+  .startsWith('https://', `URL must start with [${urlStart}].`)
```

Comment:
```suggestion
  .startsWith(urlStart, `URL must start with [${urlStart}].`)
```

Reply:
modified in commit: 221668d5d0b826386b4476d753022859713b8b0c